packages/omg-js-rootchain/src/index.js

```diff
@@ -94,6 +95,45 @@ class RootChain {
     debug(`encoded block number: ${encodedBlkNum}`)
     return Number(this.eth.abi.decodeParameter('uint256', encodedBlkNum))
   }
+
+  async startExit (fromAddress, utxoPos, txBytes, proof, sigs, privateKey) {
+    const txDetails = {
+      from: fromAddress,
+      to: this.plasmaContractAddress,
+      data: this.plasmaContract.methods.startExit(
+        utxoPos,
+        Web3Utils.hexToBytes(`0x${txBytes}`),
+        Web3Utils.hexToBytes(`0x${proof}`),
+        Web3Utils.hexToBytes(`0x${sigs}`)
+      ).encodeABI(),
+      gas: 2000000
+    }
+
+    return sendTx(this.eth, txDetails, privateKey)
+  }
+
+  async finalizeExits (fromAddress, token, privateKey) {
+    const txDetails = {
+      from: fromAddress,
+      to: this.plasmaContractAddress,
+      data: this.plasmaContract.methods.finalizeExits(token).encodeABI(),
+      gas: 2000000
+    }
+
+    return sendTx(this.eth, txDetails, privateKey)
+  }
+}
+
+async function sendTx (eth, txDetails, privateKey) {
```

Comment:
Nice, I like how calling this without privateKey assumes that user is sending from an unlocked Geth account